src/irr/asset/COBJMeshFileLoader.cpp

```diff
@@ -80,13 +80,15 @@ asset::SAssetBundle COBJMeshFileLoader::loadAsset(io::IReadFile* _file, const as
         _override
     );
 
-	if (_params.quantNormalCache == nullptr)
+	if (_params.meshManipulatorToUse == nullptr)
 	{
-		os::Printer::log("quantNormalCache is nullptr", ELL_ERROR);
+		os::Printer::log("meshManipulatorToUse is nullptr", ELL_ERROR);
```

Comment:
if its nulll, you just take the default one from the asset manager !

Reply:
I do it here:
https://github.com/Przemog1/IrrlichtBAW/blob/shader_pipeline/include/irr/asset/IAssetManager.h#L260